aspnetcore/fundamentals/minimal-apis/7.0-samples/WebMinJson/Program.cs

```diff
@@ -44,4 +47,221 @@ class Todo
     public bool IsComplete { get; set; }
 }
 // </snippet_2>
+#elif writeasjsonasync
+var builder = WebApplication.CreateBuilder(args);
+
+var app = builder.Build();
```

Comment:
```suggestion
var app = WebApplication.Create();
```

Nit: for samples where we don't actually modify the builder before building it, we might want to uses `WebApplication.Create` API directly so users know the more direct approach.